Documentation/Conditions/Index.rst

```diff
@@ -795,73 +795,6 @@ compatVersion()
       [compatVersion("11.5.1")]
 
 
-.. index:: Conditions; loginUser
-.. _condition-function-loginUser:
-
-loginUser()
-===========
-
-..  deprecated:: 12.4
```

Comment:
keep a hint or migration information maybe?

Reply:
I would remove all that old stuff.

Reply:
Migrating Conditions has caused a lot of trouble for users in the past

Reply:
Okay okay, will leave it in somehow

Reply:
We could maybe also make a "Migration" section, we had that in the past

Reply:
It is already in place in v12. Will just adjust the note above and leave all the stuff the same.